src/System.Private.ServiceModel/tests/Common/Scenarios/ServiceInterfaces.cs

```diff
@@ -506,15 +506,3 @@ public interface IWcfAspNetCompatibleService
     string EchoTimeAndSetCookie(string name);
 }
 
-[ServiceContract]
```

Comment:
Reviewers: this moved verbatim into IUtil.cs in the Common\Infrastructure project.